agent/tool-scripts/base-tool

```diff
@@ -278,6 +282,8 @@ function usage {
 		printf -- "\t                           but not both at the same time\n"
 		;;
 	esac
+
+	printf -- "Help option - do *not* specify this when you register a tool:\n\n"
```

Comment:
😀  You don't think this goes without saying?

Reply:
You can never be sure :-)